### PR TITLE
Improve Vehicle Info UI

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Work on #483 
- Make Vehicle Info UI movable element in layout editor
- Prevent Vehicle Info UI from being hidden when opening displays (eg. ACE3 Interaction Menu)
- Add support for variable amount of intercoms and racks to Vehicle Info UI (auto-resize in height)
- Streamline ACE3 action names (length, capitalization)
- Streamline key names (capitalization)

Vehicle Info UI is now being created with `cutRsc` (like most ACE3 UIs), so it does not get hidden when opening displays. Drawback of that is that it also doesn't hide itself when pause menu is open (not really an issue).

### CODE REVIEW

File: addons/sys_intercom/fnc_intercomChildrenActions.sqf [hunk 1]
```diff
@@ -22,10 +22,10 @@ private _actions = [];
 private _intercomNames = _target getVariable [QGVAR(intercomNames), []];
 
 {
-    (_intercomNames select _forEachIndex) params ["_intercomName", "_intercomDisplayName"];
+    (_intercomNames select _forEachIndex) params ["_intercomName", "", "_intercomShortName"];
```
Comment (TheMagnetar) [inline addons/sys_intercom/fnc_intercomChildrenActions.sqf:25]: Personally I would leave as it was before.

Comment (jonpas) [reply addons/sys_intercom/fnc_intercomChildrenActions.sqf:25]: Previously it was:
```
Main Action
    Intercom
        Crew Intercom
        Pax Intercom
```
This changes it to:
```
Main Action
    Intercom
        Crew
        Pax
```
Which I think is better, it's obviously Intercom, you already select that with first action, and it takes less space.

We can of course handle that differently for better compatibility with user-defined intercom names and so on.

Comment (TheMagnetar) [reply addons/sys_intercom/fnc_intercomChildrenActions.sqf:25]: Let's keep your changes.

**Review by TheMagnetar — APPROVED** [review]
Works like a charm

**Pull request merged**